src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -79061,13 +79058,15 @@ void M(out object o)
                 Diagnostic(ErrorCode.WRN_NullReferenceReceiver, "P").WithLocation(7, 9));
         }
 
-        [Fact]
-        public void UnassignedStructAutoProperty_Constructor()
+        [Theory]
+        [InlineData("object")]
+        [InlineData("object?")]
+        public void UnassignedStructAutoProperty_Constructor(string propertyType)
```

Comment:
The logic added in NullableWalker is conditional on LangVer, so I was expecting some LangVer tests in this file. Can we add?